messaging.rst

```diff
@@ -238,7 +231,7 @@ Non cancellable, Non expirable.
 Invariants
 ^^^^^^^^^^
 
-- The :term:`balance proof` merkle tree must have the corresponding lock removed (and only this lock).
+- The :term:`balance proof` must contain the hash of pending lock list with the corresponding lock removed (and only this lock).
```

Comment:
"of pending lock list" is this correct? To me it's difficult to decide if we talk about one lock in a list or a list of locks. And I am not sure what the "corresponding lock" refers to. Maybe it can be rephrased to be a bit more clear?

Reply:
* "pending lock list":  I haven't thought about this.  People say "task list" not "tasks list".  Maybe "the list of pending locks" is clearer.
* "corresponding": I agree it's terrible now.  I'll give another try.